### PR TITLE
Fixed ClassCastException for empty Table indexes field in groovy

### PR DESCRIPTION
**Pull request by altro3**
See https://github.com/micronaut-projects/micronaut-openapi/issues/964

### CODE REVIEW

**Comment by altro3** [comment]
@dstepanov Look to this bug. Spent several hours to find it :-))))

File: data-processor/src/main/java/io/micronaut/data/processor/mappers/jpa/jx/TableAnnotationMapper.java [hunk 1]
```diff
@@ -54,14 +54,16 @@ public List<AnnotationValue<?>> map(AnnotationValue<Annotation> annotation, Visi
...
-        Optional.ofNullable((AnnotationValue<Annotation>[]) annotation.getValues().get("indexes"))
-                .ifPresent(indexes -> {
-                    final AnnotationValue<Index>[] annotationValues =
-                            (AnnotationValue<Index>[]) Arrays.stream(indexes)
-                                    .map(a -> mapper.map(a, null)
-                                    .get(0)).toArray(AnnotationValue[]::new);
-                   idxBuilder.member("value", annotationValues);
-                });
+
+        Object indexesValue = annotation.getValues().get("indexes");
```
Comment (dstepanov) [inline data-processor/src/main/java/io/micronaut/data/processor/mappers/jpa/jx/TableAnnotationMapper.java:58]: I wonder if `annotationValues` can be used here

Comment (altro3) [reply data-processor/src/main/java/io/micronaut/data/processor/mappers/jpa/jx/TableAnnotationMapper.java]: @dstepanov fixed

**Comment by altro3** [comment]
@dstepanov What about this? Maybe we can include this fix to release 3.9.7?

**Comment by dstepanov** [comment]
Yeah, thanks!

**Pull request merged**